src/phoenix/pointcloud/umap_parameters.py

```diff
@@ -0,0 +1,53 @@
+from dataclasses import dataclass
+
+DEFAULT_MIN_DIST = 0.0
+DEFAULT_N_NEIGHBORS = 30
+DEFAULT_N_SAMPLES = 500
+
+MIN_NEIGHBORS = 5
+MAX_NEIGHBORS = 100
+MIN_SAMPLES = 1
+MAX_SAMPLES = 1000
+MIN_MIN_DIST = 0.0
+MAX_MIN_DIST = 0.99
+
+
+def get_umap_parameters(default_umap_parameters):
+    return UMAPParameters(
+        min_dist=default_umap_parameters.get("minDist", DEFAULT_MIN_DIST),
+        n_neighbors=default_umap_parameters.get("nNeighbors", DEFAULT_N_NEIGHBORS),
+        n_samples=default_umap_parameters.get("nSamples", DEFAULT_N_SAMPLES),
+    )
+
+
+@dataclass
+class UMAPParameters:
+    """
+    Can include any of the three keys: "minDist", "nNeighbors", "nSamples" necessary for point cloud initialization

```

Comment:
I think it's no longer a partial now correct?

Reply:
That's right.